react/features/base/participants/functions.js

```diff
@@ -75,6 +75,39 @@ export function getAvatarURL({ avatarID, avatarURL, email, id }: {
 }
 
 /**
+ * Returns display name of the participant identified by the given ID from Redux
+ * state.
+ *
+ * @param {(Function|Object|Participant[])} stateOrGetState - The redux state
+ * features/base/participants, the (whole) redux state, or redux's
+ * {@code getState} function to be used to retrieve the
+ * features/base/participants state.
+ * @param {string} id - The participant's id for whom the display name is to be
+ * obtained.
+ * @returns {(string|undefined)}
+ */
+export function getDisplayName(stateOrGetState: Object | Function, id: string) {
```

Comment:
are these 2 functions really that useful? they are 2 lines, which can be compacted into 1...

Reply:
Moreover, I think this should probably go outside of react/, since they are helpers for legacy code